src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-weight: bold;
+  font-size: 17px;
+  margin: 0;
+}
+
+h2 {
+  font-weight: normal;
+  font-size: 15px;
+  margin: 1em 0;
+}
+
+p {
+  margin: 0;
+}
+
+.green_link {
```

Comment:
`green_link` is a bad name for the class. It's not semantic and has no meaning.
You may name the class just `link`, for example, or use `a` tag selector for this task